app/models/hydrus/rights_metadata_ds.rb

```diff
@@ -48,6 +52,16 @@ class Hydrus::RightsMetadataDS < ActiveFedora::OmDatastream
     xml.machine(type: gcode) { xml.text(code) }
   end
 
+  # just a wrapper to invalidate @dra_object
+  def content=(xml)
+    @dra_object = nil
+    super
+  end
+
+  def dra_object
```

Comment:
I think the problem is not that we are missing this object, but that the indexer should not be referencing this object.

Reply:
Don't disagree. Was shooting to put up a quick fix and move on to other fires.

Reply:
I'm fairly certain we can delete all of this: https://github.com/sul-dlss/hydrus/blob/dd3a54246e9c35a179d6c0c86dc446df47500194/app/indexers/rights_metadata_datastream_indexer.rb#L19-L62

Reply:
@jcoyne OK, I'll have a look.

Reply:
@jcoyne if I remove all the dra stuff in the rights metadata datastream indexer, it's not going to be doing very much, which has me wondering why we have all this code in Hydrus now. I'm not eager to do a deep dive on this one.

Reply:
This was added so that we could upgrade to the latest version of dor-services which does not have its own indexing.  This indexing code was for `Dor::RightsMetadataDS` not for `Hydrus::RightsMetadataDS`